src/game_inventory.cpp

```diff
@@ -117,7 +117,6 @@ int game::inv_for_filter( const std::string &title, item_filter filter,
 
 int game::inv_for_all( const std::string &title, const std::string &none_message )
 {
-    const std::string msg = ( none_message.empty() ) ? _( "Your inventory is empty." ) : none_message;
     return u.get_item_position( inv_internal( u, inventory_selector_preset(),
                                 title, -1, none_message ).get_item() );
```

Comment:
Should probably be replaced with `title, -1, msg ).get_item() );`.

Reply:
Makes sense. Thanks.